### PR TITLE
ICU-21546 Update ICU4C API reference docs for 69.1

### PR DESCRIPTION
**Pull request by sven-oly**

### CODE REVIEW

**Comment by sven-oly** [comment]
Second commit removes unneeded .zip.

**Comment by hugovdm** [comment]
This isn't currently serving at https://sven-oly.github.io/icu-docs/ - fwiw I pushed to my repo, docs are building, should soon be serving commit eb55b8c at this URL: https://hugovdm.github.io/icu-docs/

**Comment by hugovdm** [comment]
Can you merge upstream/main and edit README.md? Or are you happy if I push my merge commit and README.md update to your branch?

**Comment by sven-oly** [comment]
Hi Hugo, please go ahead and merge as you see fit.

I've update my README.md to point to ICU69.1 release. I didn't update the 4J note, however, because my branch does not yet have the ICU4J.

What do you recommend?

**Comment by sven-oly** [comment]
I think this is now resolved. https://sven-oly.github.io/icu-docs/ now is publicly available.

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 |           |   C                              |             J                    |
 |-----------|----------------------------------|----------------------------------|
-| Released  | [icu4c 68.1](./apidoc/released/icu4c) | [icu4j 69.1](./apidoc/released/icu4j) |
-| Dev       | [icu4c](./apidoc/dev/icu4c)      | [icu4j](./apidoc/dev/icu4j)      |
+| Released  | [icu4c 69.1](./apidoc/released/icu4c) | [icu4j 69.1](./apidoc/released/icu4j) |
+| Dev   | [icu4c](./apidoc/dev/icu4c)       | [icu4j](./apidoc/dev/icu4j)      |
```
Comment (markusicu) [inline README.md:13]: It would be nice to revert this line to exactly what it was before.

**Review by markusicu — COMMENTED** [review]
rslgtm please squash

File: README.md (hunk 1 above)
Comment (sven-oly) [reply README.md]: Done.

**Review by hugovdm — COMMENTED** [review]
FYI: one can also squash to a single commit by simply resetting the branch head to the commit where one wants to squash, then commit all the changes together / at once.

**Pull request closed without merging**